df.raws.xml

```diff
@@ -25,6 +25,20 @@
         </virtual-methods>
     </class-type>
 
+    <class-type type-name='soundst'>
+        <stl-string name='token'/>
+        <int32_t name='index'/>
+        <stl-vector name='current_definition'>
+            <stl-string/>
+        </stl-vector>
```

Comment:
We're also missing a `<df-flagarray name='flag'/>` here.

Reply:
Did I put it in the right place?